rector.php

```diff
@@ -42,12 +43,12 @@
 
     $parameters->set(Option::SKIP, [
         // skip because of phpdocs which get mangled https://github.com/rectorphp/rector/issues/4691
-        'redaxo/src/core/lib/fragment.php',
-        'redaxo/src/core/lib/list.php',
-        'redaxo/src/core/lib/packages/manager.php',
-        'redaxo/src/core/lib/sql/sql.php',
-        'redaxo/src/core/lib/var/var.php',
-        'redaxo/src/core/lib/util/version.php',
+        // 'redaxo/src/core/lib/fragment.php',
+        // 'redaxo/src/core/lib/list.php',
+        // 'redaxo/src/core/lib/packages/manager.php',
+        // 'redaxo/src/core/lib/sql/sql.php',
+        // 'redaxo/src/core/lib/var/var.php',
+        // 'redaxo/src/core/lib/util/version.php',
```

Comment:
Die Regeln, die ich heute aktiviert habe, möchte ich eigentlich auch gerne in den Dateien anwenden. Deswegen diesen globalen Ausschluss vorerst auskommentiert.

Reply:
da das issue (s.o.) was grundlage war - warum diese dateien ge-skipped wurden - gefixed ist, könnte man die ggf. auskommentierte skip liste auch komplett rauswerfen (inkl dem kommentar)?